### ISSUE TITLE
inaccessible links in Social media links

### ISSUE THREAD

**Issue by PurpurTentakel97** [TechStreamConference/website-frontend#239]
The hover for the Link status is not accessible.
Consider so add the tooltip to the aria-label a well.
